install.yml

```diff
@@ -65,19 +94,18 @@
     - name: set correct file permission for everything in the deployment
       file:
         path: "{{ root_path }}"
-        group: "{{ ngi_group }}"
+        group: "{{ ngi_sw_group }}"
         state: directory
         recurse: yes
         mode: g+rwX,o=rX
 
-    - name: Make wildwest directory editable by anyone that can access miarka
+    - name: make containers directory readable by anyone that can access miarka
       file:
-        path: "{{ proj_root }}"
-        group: "{{ ngi_group }}"
+        path: "{{ ngi_containers }}"
+        group: "{{ ngi_sw_group }}"
         state: directory
         recurse: yes
-        mode: g+rwXs,o=rwX
-      when: deployment_environment == "staging"
+        mode: g+rwX,o=rX
```

Comment:
I see that the 's' flag is removed from the recursive mode assignment, and this 's' flag seems to be applied only on the directory root folder. Is there perhaps any chance that there could be an already existing sub-folder that doesn't have this setting?

Reply:
this is a great point, right now, there is indeed some existing sub-folders and I intend to deal with them when this is to be deployed and synced for the first time, either by (re-)moving offending folders or asking Uppmax to update ownership/permissions. thereafter, it is expected that everything is either only added through this playbook + sync or will inherit the root folder settings + effective umask of user (the plan is that Uppmax will set a global umask). We were also discussing options for ensuring the consistency of permissions and ownership in the projects jus this morning..